github/table_github_audit_log.go

```diff
@@ -0,0 +1,146 @@
+package github
+
+import (
+	"context"
+	"time"
+
+	"github.com/google/go-github/v45/github"
+	pb "github.com/turbot/steampipe-plugin-sdk/v3/grpc/proto"
```

Comment:
```suggestion
	"github.com/turbot/steampipe-plugin-sdk/v3/grpc/proto"
```

I believe we should be able to get rid of the alias here, and then for references below, instead of `pb`, we could use `proto`.